cpa/parsing.go

```diff
@@ -100,9 +100,9 @@ func parseBundle(bundle map[string]string, rules ...LintRule) (*Policy, error) {
 			continue
 		}
 
-		moduleMap[file] = mod
+		moduleMap[name] = mod
+		source[name] = rego
```

Comment:
is source = conent?
If yes, can we call this content then, instead of introducing another name for content?
Also, nit, contentMap, similar to moduleMap?

Reply:
Nevermind, I see some usage of word `source` already existing in code.